raiden_contracts/contracts/EndpointRegistry.sol

```diff
@@ -1,11 +1,8 @@
-/*
- * This contract is a registry which maps the Ethereum Address to their
- *  endpoint i.e sockets.
- * The Ethereum address registers his address in this registry.
-*/
-
 pragma solidity ^0.4.23;
 
+/// @title Endpoint Registry
+/// @notice This contract is a registry which maps the Ethereum Address to their
```

Comment:
which maps **an** Ethereum address to **its** endpoint, **i.e.** sockets. The Ethereum address registers **its** address in this registry.